### PR TITLE
Add storage integration test

### PR DESCRIPTION
**Pull request by fuku2014**
### Summary

- Add storage integration test

### Test

- [x] `make integration` succeed 🍋

### CODE REVIEW

File: service/internal/integrationtest/config.go [hunk 1]
```diff
@@ -25,3 +25,25 @@ func LoadConfigWithDefaultRegion(defaultRegion string) nifcloud.Config {
...
 }
+
+// LoadConfigWithStorageDefaultRegion loads the default configuration for the storage SDK,
+// and falls back to a default region if one is not specified.
+func LoadConfigWithStorageDefaultRegion(defaultRegion string) nifcloud.Config {
+	var lm aws.ClientLogMode
+
+	if strings.EqualFold(os.Getenv("NIFCLOUD_DEBUG_REQUEST"), "true") {
+		lm |= aws.LogRequest
+
```
Comment (aokumasan) [inline service/internal/integrationtest/config.go:36]: I think this line break is unnecessary 🥺

Comment (fuku2014) [reply service/internal/integrationtest/config.go]: 🥺🥺🥺  

Fixed!!

**Review by aokumasan — APPROVED** [review]

File: service/internal/integrationtest/config.go (hunk 1 above)
Comment (aokumasan) [reply service/internal/integrationtest/config.go]: Thanks!! 👍

**Comment by aokumasan** [comment]
```
Integration Testing SDK clients
go test -timeout=10m -count=1 -tags "integration" -v -run '^TestInteg_' ./service/...
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/computing   0.003s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/computing/internal/endpoints        0.009s [no tests to run]
?       github.com/nifcloud/nifcloud-sdk-go/service/computing/types     [no test files]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/dns 0.015s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/dns/internal/endpoints      0.022s [no tests to run]
?       github.com/nifcloud/nifcloud-sdk-go/service/dns/types   [no test files]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/ess 0.005s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/ess/internal/endpoints      0.005s [no tests to run]
?       github.com/nifcloud/nifcloud-sdk-go/service/ess/types   [no test files]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/hatoba      0.004s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/hatoba/internal/endpoints   0.004s [no tests to run]
?       github.com/nifcloud/nifcloud-sdk-go/service/hatoba/types        [no test files]
?       github.com/nifcloud/nifcloud-sdk-go/service/internal/accept-encoding    [no test files]
?       github.com/nifcloud/nifcloud-sdk-go/service/internal/integrationtest    [no test files]
=== RUN   TestInteg_00_DescribeRegions
--- PASS: TestInteg_00_DescribeRegions (0.65s)
=== RUN   TestInteg_01_NiftyDescribeElasticLoadBalancers
--- PASS: TestInteg_01_NiftyDescribeElasticLoadBalancers (4.67s)
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/internal/integrationtest/computing  5.327s
=== RUN   TestInteg_00_ListHostedZones
--- PASS: TestInteg_00_ListHostedZones (0.42s)
=== RUN   TestInteg_01_GetHostedZone
--- PASS: TestInteg_01_GetHostedZone (0.37s)
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/internal/integrationtest/dns        0.792s
=== RUN   TestInteg_00_ListIdentities
--- PASS: TestInteg_00_ListIdentities (0.10s)
=== RUN   TestInteg_01_GetSendQuota
--- PASS: TestInteg_01_GetSendQuota (0.07s)
=== RUN   TestInteg_02_GetSendStatistics
--- PASS: TestInteg_02_GetSendStatistics (0.06s)
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/internal/integrationtest/ess        0.229s
=== RUN   TestInteg_00_ListClusters
--- PASS: TestInteg_00_ListClusters (0.32s)
=== RUN   TestInteg_01_GetCluster
--- PASS: TestInteg_01_GetCluster (0.31s)
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/internal/integrationtest/hatoba     0.641s
=== RUN   TestInteg_00_DescribeNASSecurityGroups
--- PASS: TestInteg_00_DescribeNASSecurityGroups (0.08s)
=== RUN   TestInteg_01_DescribeNASInstances
--- PASS: TestInteg_01_DescribeNASInstances (0.05s)
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/internal/integrationtest/nas        0.142s
=== RUN   TestInteg_00_DescribeDBEngineVersions
--- PASS: TestInteg_00_DescribeDBEngineVersions (0.09s)
=== RUN   TestInteg_01_DescribeDBInstances
--- PASS: TestInteg_01_DescribeDBInstances (0.06s)
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/internal/integrationtest/rdb        0.151s
=== RUN   TestInteg_00_GetService
--- PASS: TestInteg_00_GetService (0.09s)
=== RUN   TestInteg_01_GetBucket
--- PASS: TestInteg_01_GetBucket (0.03s)
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/internal/integrationtest/storage    0.122s
?       github.com/nifcloud/nifcloud-sdk-go/service/internal/s3shared   [no test files]
?       github.com/nifcloud/nifcloud-sdk-go/service/internal/s3shared/arn       [no test files]
?       github.com/nifcloud/nifcloud-sdk-go/service/internal/s3shared/config    [no test files]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/nas 0.003s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/nas/internal/endpoints      0.003s [no tests to run]
?       github.com/nifcloud/nifcloud-sdk-go/service/nas/types   [no test files]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/rdb 0.003s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/rdb/internal/endpoints      0.010s [no tests to run]
?       github.com/nifcloud/nifcloud-sdk-go/service/rdb/types   [no test files]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/storage     0.009s [no tests to run]
?       github.com/nifcloud/nifcloud-sdk-go/service/storage/internal/arn        [no test files]
?       github.com/nifcloud/nifcloud-sdk-go/service/storage/internal/customizations     [no test files]
testing: warning: no tests to run
PASS
ok      github.com/nifcloud/nifcloud-sdk-go/service/storage/internal/endpoints  0.003s [no tests to run]
?       github.com/nifcloud/nifcloud-sdk-go/service/storage/types       [no test files]
```

**Comment by aokumasan** [comment]
LGTM 💯 🍋

**Pull request merged**